File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service("brandService",function($http){
	
	
//	添加模板数据时需要的品牌数据 要求返回的格式：[{id:1,text:"SSS"},{}...]
	this.findBrandList=function(){
		return $http.get("../brand/findBrandList");
	}
	
	 this.findPage=function(pageNo,pageSize){
		 return $http.get("../brand/findPage/"+pageNo+"/"+pageSize);
	 }
	 
	 this.findAll=function(){
		 return $http.get("../brand/findAll");
	 }
	 
	 
	 this.add=function(entity){
		 return $http.post("../brand/add",entity);
	 }
	 this.update=function(entity){
		 return  $http.post("../brand/update",entity);
	 }
	 
	 
	 this.findOne=function(id){
		 return $http.get("../brand/findOne/"+id);
	 }
	 
	 this.dele=function(selectIds){
		 return $http.get("../brand/dele/"+selectIds);
	 }
	 
	 this.search=function(pageNo,pageSize,searchEntity){
 		return $http.post("../brand/search/"+pageNo+"/"+pageSize,searchEntity);
 	 }
	
})

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {
	
	public List<TbBrand> findAll();

	public PageResult findPage(int pageNo, int pageSize);

	public void add(TbBrand brand);

	public TbBrand findOne(Long id);

	public void update(TbBrand brand);

	public void dele(Long[] ids);

	public PageResult findPage(int pageNo, int pageSize, TbBrand brand);

	public List<Map> findBrandList();
	
}
